modules/nix/nix-optimise.nix

```diff
@@ -0,0 +1,30 @@
+{ config, lib, ...}:
+with lib;
+
+let
+  cfg = config.nix.optimise;
+in
+
+{
+  options = {
+    nix.optimise = {
+      enable = mkOption {
```

Comment:
Please rename this to `automatic` to match [the upstream NixOS module](https://github.com/NixOS/nixpkgs/blob/29f96cc801de8e2efd80f76ca80a389f548d3595/nixos/modules/services/misc/nix-optimise.nix#L10), so that people can share configuration between their nix-darwin and NixOS systems.